src/org/omegat/Bundle.properties

```diff
@@ -2543,6 +2541,7 @@ COLOR_MARK_ALT_TRANSLATION=Alternative translation highlight
 GUI_COLORS_COLOR=Item:
 GUI_COLORS_RESET_COLOR=&Reset Colour
 PREFS_TITLE_COLORS=Colours
+PREFS_COLOR_VALUE_PARSE_ERROR=Cannot set custom color for {0}, default to {1}.
 
```

Comment:
I'm changing "color" to "colour" and proposing a tweak the phrasing.

Reply:
thank you.